capsules/src/process_console.rs

```diff
@@ -17,6 +17,7 @@
 //!  - 'panic' causes the kernel to run the panic handler
 //!  - 'process n' prints the memory map of process with name n
 //!  - 'kernel' prints the kernel memory map
+//!  - 'Up Arrow' and 'Down Arrow' scroll through the command history
```

Comment:
Edit documentation to add the commands history length.

Reply:
Added in the Process_Console.md